docs/contributing/screenshot-testing.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Screenshot Testing
+---
+
+## What is the Screenshot Testing
+Screenshot test is visual test that takes a screenshot, then compares it to a reference screenshot file saved. When ui changes, one of the major considerations is whether visual changes exist. Some changes may be easy to check, but some changes are hard to check due to very small change. With Screenshot test, very small changes can easily check due to compare pixel-by-pixel. Additionally, Screenshot testing is the most reliable test for static images.
+
+## Prerequisites
+Clone sandstone from GitHub, install dependencies and connect the modules using Lerna:
+
+```shell
+# clone the repo!
+git clone git@github.com:enactjs/sandstone.git
+# move in
+cd sandstone
+# we're using git flow so develop is our working branch
+git checkout develop
+# install lerna
+npm install
+# link dependencies
+enact link
```

Comment:
enact link is optional step to develop enact other lib (enact/core, enact/ui, ui-test-utils...). Let's remove it.

```suggestion
```